docs/docs/guides/Web3_Utils_module/Mastering_web3-utils.md

```diff
@@ -0,0 +1,226 @@
+# Mastering web3.js Utils
+
+## Introduction
+
+## Imports
+
+There are three different way to import utils package.
+
+1. Import the entire web3 module and initialize a provider:
+
+    `const [ Web3 ] = require("web3"); ` 
+
+    then inialiazing a provider with 
+    `const web3 = new Web3("");`
+
+    To be able to access the package you need to type
+    `web3.utils.toWei()`
+
+
+2. Import only the utils package from web3:
+
+    `const { utils } = require("web3");` 
```

Comment:
it will be great to have imports TypeScript style also.  You can use tabs or just write 2 options near
```ts 
import { Web3 } from 'web3'
```

to import only utils package:
```ts 
import * as utils from 'web3-utils';
```